### DISCUSSION TITLE
Add pictures to comments.

### DISCUSSION THREAD

**Discussion by Mymko** [PHPOffice/PhpSpreadsheet#2186] [Ideas]
Hello.
Are there any hopes that [this idea](https://github.com/PHPOffice/PhpSpreadsheet/issues/1547) will be implemented?)

**Comment by MarkBaker** [comment]
Comments currently needs a rewrite to support the changes that MS have implemented for threaded comments, and I'll be looking at that over the Summer, so I ___may___ look at adding comment background images as part of that work.
